Cargo.toml

```diff
@@ -54,3 +54,10 @@ env_logger = "0.9.0"
 
 [build-dependencies]
 tonic-build = {version = "0.7.2",features = ["prost", "compression"]}
+
+[workspace]
+
+# dialdbg exists as a member of the workspace so it gets its own crate.
+members = [
+    "dialdbg",
```

Comment:
can we call this `viam-dialdbg` so that you can more easily predict the `cargo run -p` invocation?

Reply:
Interesting. `cargo run -p` outputs:
```
Possible packages/workspace members:
    viam-dialdbg
    viam-rust-utils
```
I _could_ change the workspace member name to be `viam-dialdbg` and change the directory to be called `viam-dialdbg`, but I'm not sure how idiomatic that is. I think we have a difference between directory name and package name here. `rust-utils` is the directory/repo, but the package is called `viam-rust-utils`. Similarly, I think `dialdbg` should be the directory/workspace name, and the package should be called `viam-dialdbg`. What are your thoughts, @stuqdog?

Reply:
eh i dont feel strongly!

Reply:
For consistency's sake/to avoid modifying the existing `rust-utils` structure, I think I favor keeping it as-is, but it's not a strong preference!